### ISSUE TITLE
[Legitimate Site Blocked] www.metaosk.com

### ISSUE THREAD

**Issue by Hickshu** [MetaMask/eth-phishing-detect#7138]
https://www.metaosk.com/

Hi,Is this a healthy and SSL certified website so that we can open it without being restricted?

Now, whenever a browser with Metamask software installed is opened, a warning message pops up, which makes many people afraid to open our website.

Hope we can get a solution, thank you!

**Comment by devenblake** [comment]
Duplicate of #7122.

**Comment by Hickshu** [comment]
Devenblake Hello.
Metaosk.com is a healthy and legally certified official website, please help us so that our domain can be unblocked.

Our official website is legal and has SSL certification, but now whenever you start Metaosk.com from your computer, you will be warned that it is a phishing website, which causes many users to distrust us, so we hope that our domain can be officially whitelisted.

**Comment by Hickshu** [comment]
![1649823757949](https://user-images.githubusercontent.com/103487348/163919774-16c5bdc0-51d2-4308-851f-1b6f0307c8ec.jpg)
